File: app/src/main/java/datanapps/basicsample/ui/BasicSampleApplication.java
package datanapps.basicsample.ui;

import android.app.Application;

import datanapps.basicsample.utils.DNASharedPreferenceUtils;

public class BasicSampleApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DNASharedPreferenceUtils.init(this);
    }
}

File: app/src/main/java/datanapps/basicsample/ui/BasicSampleApplication.kt
package datanapps.basicsample.ui

import android.app.Application

import datanapps.basicsample.utils.DNASharedPreferenceUtils

class BasicSampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        DNASharedPreferenceUtils.init(this)
    }
}
